### PR TITLE
PLAT-138171: Fixed the glob patterns for test files

### PR DESCRIPTION
**Pull request by MikyungKim**
This PR fixes wrong glob patterns for the test files.

Enact-DCO-1.0-Signed-off-by: Mikyung Kim (mikyung27.kim@lge.com)

### CODE REVIEW

File: config/jest/jest.config.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ module.exports = {
 	setupFilesAfterEnv: [require.resolve('./setupTests'), userSetupFile].filter(Boolean),
 	testMatch: [
 		'<rootDir>/**/__tests__/**/*.{js,jsx,ts,tsx}',
-		'<rootDir>/**/?(*.)(spec|test).{js,jsx,ts,tsx}',
+		'<rootDir>/**/?(*.)+(spec|test).[jt]s?(x)',
```
Comment (hong6316) [inline config/jest/jest.config.js:70]: I think the previous style can be more easily understood by developers, it's my personal opinion

**Comment by Sangwook-Lee** [comment]
LGTM!

**Review by Sangwook-Lee — APPROVED** [review]

File: config/jest/jest.config.js (hunk 1 above)
Comment (MikyungKim) [reply config/jest/jest.config.js:70]: Do you mean the extension part? If someone understands the former part, (s)he will understand the changed one too.
The main purpose of this change is to fix the wrong glob pattern to the default one from jest is currently using.

**Review by hong6316 — APPROVED** [review]
LGTM

**Pull request merged**